### DISCUSSION TITLE
Region default value version changes

### DISCUSSION THREAD

**Discussion by deving1995** [bbc/sqs-consumer#377] [Help] [answered]
Hello,

I noticed that in version ^5.7.0 I did not need to specify the region of the SQS Queue for it to find the queue, but in version ^7.0.2 there is a default value of `eu-west-1` for the `region`

^5.7.0 - No default value for `region` specified
![image](https://user-images.githubusercontent.com/87441052/226063322-f403c1e1-5949-4d60-a5e7-99d6b1140029.png)

^7.0.2 - Default value specified for `region`
![image](https://user-images.githubusercontent.com/87441052/226063170-fa33c608-aae6-41c5-ac3a-8ef66fb08525.png)

Because my queue is in `us-east-1` I have to specify a region for it to work. If I don't specify a region I get this error:
![image](https://user-images.githubusercontent.com/87441052/226063527-aa95fadf-84f5-434e-a506-27e3fb6d5c1d.png)

I'm curious, how did the region get specified before, was it stripped from the queue URL itself?

At first I chalked it up to there being something wrong with my Node installation, or my SQS Client SDK installation based on the notes in the README, but it was neither of those things. Then I thought it had something to do with the consumer not finding my AWS credentials in the .env file.

**Comment by deving1995** [comment]
Ope, just found out why -> https://github.com/bbc/sqs-consumer/pull/56 I just needed the region to be named "AWS_REGION" in my .env file, I'm not sure why I didn't need that in ^5.7.0 though.

**Comment by nicholasgriffintn** [comment, marked as answer]
That's a good point, I didn't realise this, the comment should probably be changed to `process.env.AWS_REGION || "eu-west-1"`, it's used here:

https://github.com/bbc/sqs-consumer/blob/main/src/consumer.ts#L79

I'll put a fix in for that later.

As we mostly use eu-west-1 internally, it's defaulted to that.

I'm not sure exactly why that's needed now vs 5.7.0, but my only guess would be that something has changed with your application or AWS SDK V3. If you find out, please update me, I'd be interested to know :)

**Reply by nicholasgriffintn** [reply]
Opened a PR here to fix the docs: https://github.com/bbc/sqs-consumer/pull/378
